bedrock/base/templates/macros.html

```diff
@@ -205,22 +205,24 @@
       <div class="triangle-border"></div>
 
       <ul class="adjunctnav{% if activenav == 'desktop' %} active{% endif %}">
+        <li><a href="{{ url('firefox.channel.desktop') }}" data-link-type="nav" data-link-name="Side Menu: Pre-release downloads">{{ _('Pre-release downloads') }}</a>
```

Comment:
@flodolo - we've added a new string here to the family nav. Would you like it in an l10n tag, or are we good to just let this sit while we translate the pages? (Note: this is the last string we we're waiting on. I think we're good to start localizing).